src/lib/gundb/UserPropertiesClass.js

```diff
@@ -64,9 +64,9 @@ export default class UserProperties {
       let props
       const { propsNode } = this
       const { defaultProperties } = UserProperties
-
       try {
-        props = await defer(() => fromPromise(propsNode.decrypt())) // init user storage
+        //sync from storage
+        props = await defer(() => fromPromise(propsNode.then(_ => propsNode.decrypt()))) // init user storage
```

Comment:
1.
```suggestion
        props = await defer(() => fromPromise(propsNode.then(() => propsNode.decrypt()))) // init user storage
```

2. do we need to retry await propsNode.then() too ?

if not we could do just
```suggestion
        await propsNode.then()
        
        props = await defer(() => fromPromise(() => propsNode.decrypt())) // init user storage
```

Reply:
we can retry... why not.